src/auth/jwt.rs

```diff
@@ -61,17 +66,18 @@ impl JWT {
     ///
     /// # Example
     /// ```rust
+    /// use serde_json::Map;
```

Comment:
Should I hide this import in the doctest or not?
```suggestion
    /// #use serde_json::Map;
```

Reply:
I wouldn't hide it. I think it makes clear to the user what to import to use this feature.